tests/unit/pillar_test.py

```diff
@@ -130,38 +130,6 @@ def test_topfile_order(self, Matcher, get_file_client):
         pillar = salt.pillar.Pillar(opts, grains, 'mocked-minion', 'base')
         self.assertEqual(pillar.compile_pillar()['ssh'], 'foo')
 
-    @patch('salt.pillar.salt.fileclient.get_file_client', autospec=True)
```

Comment:
``merge_lists=True`` doesn't seem to be tested here or did I missed something out?